projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/classic-dark/colors.scss

```diff
@@ -1,11 +1,14 @@
+// make use of color studio variables for consistency across products
+@import 'node_modules/@automattic/color-studio/dist/color-variables.scss';
+
 // core variables (core _variables.scss)
```

Comment:
Classic Dark is an outlier as the colors are taken from wp-admin directly. So color studio colors only apply to a few variables.